kubernetes/plugin.go

```diff
@@ -127,6 +129,14 @@ func listK8sDynamicCRDs(ctx context.Context, cn *connection.ConnectionCache, c *
 	}
 
 	crds := []v1.CustomResourceDefinition{}
+	temp_crds := []string{}
```

Comment:
We should add a step if required to sort the CRDs by their fully qualified names to ensure that we always generate a table with the singular spec name for the first item alphabetically as per our design

Reply:
Add a step to sort the CRDs list